ga4gh/datamodel/variants.py

```diff
@@ -787,9 +787,20 @@ def buildMetadata(
         return ret
 
     def isAnnotated(self, path):
+        # assumes that all files in the directory look like the first
+        return self.hasAnnField(path) or self.hasConsequenceField(path)
+
+    def hasAnnField(self, path):
+        pysamreader = self.openFile(
+            glob.glob(os.path.join(path, "*.vcf.gz"))[0])
+        items = [x[0] for x in pysamreader.header.info.items()]
+        return 'ANN' in items
+
+    def hasConsequenceField(self, path):
         pysamreader = self.openFile(
             glob.glob(os.path.join(path, "*.vcf.gz"))[0])
```

Comment:
The code for obtaining an open VCF is repeated multiple times.  Put in function rather than copy and paste.

Is the name of the file already know by this point?  It seems odd to glob for it multiple times.  Also, this if the file is missing this will result in a very confusing IndexError rather than report the file missing.